### PR TITLE
resolve code duplicate issue

### PR DESCRIPTION
**Pull request by KhudaDad414**
**Description**
SonarClould check fails for the duplicate code in the open PRs:
- https://github.com/asyncapi/studio/pull/784/checks?check_run_id=17432376236
- https://github.com/asyncapi/studio/pull/618#issuecomment-1748713981

### CODE REVIEW

**Comment by KhudaDad414** [comment]
@fmvilas 👁️ 🙏

File: apps/studio/src/components/Navigation.tsx [hunk 1]
```diff
@@ -67,6 +68,142 @@ const ServersNavigation: React.FunctionComponent<NavigationSectionProps> = ({
...
+        }
+      >
+        Channels
+      </div>
+      <ul>{channels}</ul>
+    </>
+  );
+};
+
+const V3OperationsNavigation: React.FunctionComponent<NavigationSectionProps> = ({
```
Comment (Amzani) [inline apps/studio/src/components/Navigation.tsx:123]: @KhudaDad414 we will have more and more duplicated code as we are in the transition period to V3. 
We should just add an exception in SonarCloud, and clean after ending support for V2.
Not sure if there is an other clean way to do it.

Comment (Amzani) [inline apps/studio/src/components/Navigation.tsx:123]: Naming suggestion, I find `OperationNavigationV3` more readable than `V3OperationsNavigation`

**Review by Amzani — CHANGES_REQUESTED** [review]

**Comment by KhudaDad414** [comment]
#821 resolves the issue.

**Pull request closed without merging**